src/astrid_sedal_ex/ex05/bounded_sim.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Astrid Hæve Sedal'
+__email__ = 'astrised@nmbu.no'
+
+from src.astrid_sedal_ex.ex05.walker_sim import Walker, Simulation
+
+
+class BoundedWalker(Walker):
+
+    def __init__(self, start, home, left_limit, right_limit):
+        """
+        Initialise the walker
+
+        Arguments
+        ---------
+        start : int
+            The walker's initial position
+        home : int
+            The walk ends when the walker reaches home
+        left_limit : int
+            The left boundary of walker movement
+        right_limit : int
+            The right boundary  of walker movement
+        """
+        super().__init__(start, home)
+        self.left_limit = left_limit
+        self.right_limit = right_limit
+
+    def move(self):
+        super().move()
+
+        if super().get_position() > self.right_limit:
```

Comment:
`self.get_position()` has the same behavior here.
The `super` keyword is just for you to separate which function to call from (from the class itself or from the parent class __when__ this class is overriding a function from the parent class).

In short,
`super().move()` is for the program to understand to call `move` from parent, because this class also have `move` with different behavior.